mava/systems/jax/ippo/system.py

```diff
@@ -94,3 +94,71 @@ def design(self) -> Tuple[DesignSpec, Any]:
             component_dependency_guardrails=ComponentDependencyGuardrails,
         )
         return system, default_params
+
+
+class IPPOSystemHuber(System):
```

Comment:
We can probably remove this system and just add the Huber component in the example.

Reply:
Removed.